cli/src/pcluster/validators/ec2_validators.py

```diff
@@ -213,25 +213,40 @@ def capacity_reservation_matches_instance(capacity_reservation: Dict, instance_t
     ] == AWSApi.instance().ec2.get_subnet_avail_zone(subnet)
 
 
+def capacity_reservation_resource_group_is_service_linked_group(capacity_reservation_resource_group_arn: str):
+    try:
+        return AWSApi.instance().resource_groups.get_group_configuration(group=capacity_reservation_resource_group_arn)
+    except AWSClientError:
+        return False
```

Comment:
Can you also check the type of the configuration is `CapacityReservationPool`? See https://docs.aws.amazon.com/ARG/latest/APIReference/about-slg.html#about-slg-types-ec2-capacityreservation

Since we are adding an boto3 call, I would make the most from the call :)

Reply:
updated!